implementation/src/main/java/io/smallrye/jwt/KeyUtils.java

```diff
@@ -46,42 +51,58 @@
  */
 public class KeyUtils {
     private static final String RSA = "RSA";
+    private static final String EC = "EC";
 
-    public static PrivateKey readPrivateKey(String pemResName) throws IOException, GeneralSecurityException {
+    public static PrivateKey readPrivateKey(String pemResName, SignatureAlgorithm algo)
```

Comment:
@aaronanderson please retain all the original signatures and delegate to the overloaded ones with the RS256. Otherwise some code, starting from the test code in Thorntail, will fail to compile